src/raw-value.ts

```diff
@@ -126,64 +78,136 @@ function createReverseShadowTarget(target: ReverseProxyTarget): ReverseShadowTar
     return shadowTarget;
 }
 
+// equivalent to Object.getOwnPropertyDescriptor, but looks into the whole proto chain
+function getPropertyDescriptor(o: any, p: PropertyKey): PropertyDescriptor | undefined {
+    do {
+        const d = ReflectGetOwnPropertyDescriptor(o, p);
+        if (!isUndefined(d)) {
+            ReflectSetPrototypeOf(d, null);
+            return d;
+        }
+        o = ReflectGetPrototypeOf(o);
+    } while (o !== null);
+    return undefined;
+}
+
 export function reverseProxyFactory(env: MembraneBroker) {
 
-    function getReverseDescriptor(descriptor: PropertyDescriptor): PropertyDescriptor {
-        const reverseDescriptor = assign(ObjectCreate(null), descriptor);
-        const { value, get, set } = reverseDescriptor;
-        if ('writable' in reverseDescriptor) {
+    function getRawDescriptor(secDesc: PropertyDescriptor): PropertyDescriptor {
```

Comment:
☝️ Maybe `getRawFullDescriptor` and 👇 `hasOwnProperty` checks instead of `in` per earlier discussion.